plugins/modules/swim_intent.py

```diff
@@ -1362,6 +1379,125 @@ def get_diff_merged(self, config):
+                                has been verified.""".format(name)
+                        self.log(self.msg, "INFO")
+                    else:
+                        self.msg = """The requested image '{0}' has been un-tagged as golden in the Cisco Catalyst Center and
+                                image status has been verified.""".format(name)
+                        self.log(self.msg, "INFO")
+            else:
+                self.log("""Mismatch between the playbook input for tagging/un-tagging image as golden and the Cisco Catalyst Center indicates that
+                         the tagging/un-tagging task was not executed successfully.""", "INFO")
+
+        if self.want.get("distribution_details"):
+            image_id = self.have.get("distribution_image_id")
+
+            if self.have.get("distribution_device_id"):
+                if self.single_device_distribution:
+                    self.msg = """The requested image with device id '{0}' has been distributed successfully in the Cisco Catalyst Center and
+                             image status has been verified.""".format(self.have.get("distribution_device_id"))
+                    self.log(self.msg, "INFO")
+                else:
+                    self.log("""Mismatch between the playbook input for distributing image to device with id '{0}' and the Cisco Catalyst Center indicates that
+                         the distribution task was not executed successfully.""".format(self.have.get("distribution_device_id")), "INFO")
+            elif self.complete_successful_distribution:
+                self.msg = """The requested image with id '{0}' distributed successfully on all devices in that particular site
+                         in the Cisco Catalyst Center.""".format(image_id)
+                self.log(self.msg, "INFO")
+            elif self.partial_successful_distribution:
+                self.msg = "The requested image with id '{0}' distributed partially on some devices in the Cisco Catalyst Center.".format(image_id)
+                self.log(self.msg, "INFO")
+            else:
+                self.msg = "The requested image with id '{0}' failed to distributed on devices in the Cisco Catalyst Center.".format(image_id)
+                self.log(self.msg, "INFO")
+
+        if self.want.get("activation_details"):
+            image_id = self.have.get("activation_image_id")
+
+            if self.have.get("activation_device_id"):
+                if self.single_device_activation:
+                    self.msg = """The requested image with device id '{0}' has been activated successfully in the Cisco Catalyst Center and
+                             image status has been verified.""".format(self.have.get("activation_device_id"))
+                    self.log(self.msg, "INFO")
+                else:
+                    self.log("""Mismatch between the playbook input for activating image to device with id '{0}' and the Cisco Catalyst Center indicates that
```

Comment:
print image name??

Mismatch between the playbook's input for activating the image on the device with ID '{0}' and the actual state in the Cisco Catalyst Center suggests that the activation task might not have been executed successfully.".format(self.have.get("activation_device_id"))

Reply:
added the image name.